config/cdash.php

```diff
@@ -67,7 +67,9 @@
     'notify_pull_request' => env('NOTIFY_PULL_REQUEST', false),
     'php_exe' => env('PHP_EXE', ''),
     'phpunit_exe' => env('PHPUNIT_EXE', ''),
+    'queue_timeout' => env('QUEUE_TIMEOUT', 2000),
     'remote_workers' => env('REMOTE_WORKERS', false),
+    'retry_base' => env('QUEUE_RETRY_BASE', 5),
```

Comment:
Should this be added to `.env.example` as well?

Reply:
Unless you feel otherwise, I was inclined to leave it out. For now, it only exists to keep the `donehandler` test from taking a long time to complete.

Reply:
In my opinion, all of the possible configuration options should be documented in `.env.example` in case someone goes looking for settings to change if they experience issues.  

As of now, `.env.example` is the most complete listing of all of the CDash configuration options.  In the future, perhaps `.env.example` should be a subset of the most common options, with a larger list stored somewhere else.